pkg/workflow/tasks/discover.go

```diff
@@ -290,25 +288,25 @@ func NewViewTaskDiscover(pd *packages.PackageDiscover, cli client.Client, cfg *r
 }
 
 // GetSuspendStepDurationWaiting get suspend step wait duration
-func GetSuspendStepDurationWaiting(step v1beta1.WorkflowStep) (bool, builtintime.Duration, error) {
+func GetSuspendStepDurationWaiting(step v1beta1.WorkflowStep) (time.Duration, error) {
```

Comment:
why we omit the bool return value?

Reply:
The bool value is redundant, `time.Duration(0)` can replace it.